### ISSUE TITLE
Show/Hide Columns option

### ISSUE THREAD

**Issue by manuelglez86** [ampache/ampache#2216]
### Describe the bug

The option enable/disable the columns you wanted to see is missing, Is this on purpose? Because I think the posibility to select the columns you liked to see is very usefull.

Thanks

## Environment

* Ampache version: 4.0.3
* Web server + version:
* Server operating system:
* Client operating system:

### Client type

* Web
* Ampache

**Comment by AshotN** [comment]
As a temporary fix you can edit Ampache's CSS files to hide whatever you like.

**Comment by manuelglez86** [comment]
Thank you, if this wiil be implemented later I will wait, Im testing v4 and wont be realeasing it for our University Radio station until it'll be more stable

**Comment by lachlan-00** [comment]
i didn't remove this feature so i'll have to find out why it's gone.

**Comment by lachlan-00** [comment]
Ah, this WAS removed by me. the jquery-mediatable JS was removed due to it's age/unsupportedness.

**Comment by manuelglez86** [comment]
Dont worry, If its posible to implement it again please it would be great

Thank you.

**Comment by AshotN** [comment]
If you want a setting thing hides it, the simplest solution I can think is `display: none` being injected when a column is hidden. I'd fix it, but I'd rather focus on the new interface than updating the old one.

**Comment by lachlan-00** [comment]
mediatable was removed due to some wicked bugs in it causing the interface needing a reload.

I've put some css in for the menu but I think a db pref would work for this.

Once i get 4 stable again i'll be moving onto new dev.

**Comment by manuelglez86** [comment]
Thank you, I think its good to have an option like this where you can select the info you want to see (track name, album, artist, composer, etc), this is present in audio players like Itunes, winamp, VLC. 

I really like what you have done with ampache, it has better performance and I like the improvements  and the new error Messages of the API.

I need to make some changes to my API Client in order to handle the new success and error messages, and then I will continue testing the API.

Thanks for your hard work.

**Comment by manuelglez86** [comment]
Hi

If its posibble to show the column "comment" on the songs tables, I wil need this just for my ampache implementation, I´ve track down to this file on templates: `show_songs.inc.php` I´ve found where to add the column, but I wasnt able to track down where the data is filled. I dont undestando how is it done and dont know what I have to modify.

Thanks

**Comment by lachlan-00** [comment]
Comment is there in song_data I think.

I can add that no problem

**Comment by lachlan-00** [comment]
It might be a good opportunity to do a list option for columns like the favorites plugin

![Screenshot_20211105-221005_Firefox.jpg](https://user-images.githubusercontent.com/1305249/140508271-2837830d-41b0-467d-9a0a-5787c7eb60f3.jpg)

Song data joining might be slower but like private cataloges I think choice is fine.

**Comment by manuelglez86** [comment]
Yes it would be great that we can add or remove columns like in Itunes, it would be great that columns can be add or remove also in playlists
